spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
+
+RSpec.describe Wakes::GoogleAnalyticsApiWrapper do
+  describe '#get_pageviews_for_path' do
+    # Describes this view: https://www.google.com/analytics/web/?hl=en#report/content-pages/a1853263w3269662p8231065/%3F_u.date00%3D20140101%26_u.date01%3D20141231%26explorer-table.plotKeys%3D%5B%5D%26explorer-table.advFilter%3D%5B%5B0%2C%22analytics.pagePath%22%2C%22RE%22%2C%22%5E%2Fabout(%5C%5C%3F%7C%24)%22%2C0%5D%5D%26explorer-table.rowCount%3D100/
+    it 'returns the total pageviews for the given date range' do
+      pageviews = subject.get_pageviews_for_path('/about', :start_date => '2014-01-01', :end_date => '2014-12-31')
+
+      expect(pageviews).to eq 78_891
+    end
+
+    describe 'handles lang params' do
+      it 'only matches if original path has lang' do
+        pageviews = subject.get_pageviews_for_path('/sermons/for-his-sake-and-for-your-joy-go-low?lang=es',
```

Comment:
@joeosburn we are NOT currently using VCR here, and we ARE doing live api lookups here. Its worked out okay so far, but will probably eventually result in brittle tests and should be revisited.

My concern was that a VCR cassette must end up storing, however obscurred, something with our private key in it (even if its just a hash). I'm not too sure how obscure it would be. @pjmelling and I started discussing one day, and he seemed tentatively okay with using VCR, but we ran out of time to finish the discussion. 

If ya'll ( @joeosburn @pjmelling @gaganawhad ) are happy with me using VCR here, then I will gladly. I just don't want to accidentally commit sensitive information.